tests/fertiscan/db/update_inspection/test_upsert_inspection.py

```diff
@@ -49,8 +50,8 @@ def tearDown(self):
     def test_insert_new_inspection_minimal_fields(self):
         # Insert a new inspection with minimal required fields
         self.cursor.execute(
-            'SELECT upsert_inspection(%s, %s, %s, %s, %s, %s);',
-            (None, self.label_info_id, self.inspector_id, None, None, False),
+            "SELECT upsert_inspection(%s, %s, %s, %s, %s, %s, %s);",
```

Comment:
I suggest using `sub_label.new_inpection()` when you want to create an inspection. 
Also, an inspection is never upserted in our workflow

Reply:
These will be fixed in https://github.com/ai-cfia/ailab-datastore/issues/158.